administrate-field-lat_lng.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'administrate', '>= 0.2.1', '<= 0.3.0'
+  spec.add_dependency 'administrate', '>= 0.2.1', '<= 0.4.0'
```

Comment:
Can we make this say '< 0.5' instead of '<= 0.4.0' ? That way it will allow bugfix releases to 0.4 without needing updating again.